jsk_interactive_markers/jsk_interactive_marker/src/marker_6dof.cpp

```diff
@@ -242,7 +244,9 @@ class Marker6DOF {
 
   void publishTF(const geometry_msgs::PoseStamped& pose) {
     tf::Transform transform;
-    tf::poseMsgToTF(pose.pose, transform);
+    geometry_msgs::PoseStamped transformed_pose;
+    tf_listener_->transformPose(frame_id_, ros::Time(0), pose, pose.header.frame_id, transformed_pose);
+    tf::poseMsgToTF(transformed_pose.pose, transform);
```

Comment:
I see, you appoint the `/head_to_camera_frame/feedback` is published with frame_id which is set on rviz, right?